devdocs/settingUp.md

```diff
@@ -43,10 +43,9 @@ Important: When a version is specified, please install that version instead of t
     * HTML syntax: We prefer not to use the HTML syntax validator provided by Eclipse.
     To turn it off, go to `Window → Preferences → Validation → HTML Syntax Validator` and uncheck the `Build` option.
 3. Create main config files {These are not under revision control because their 
-   content vary from developer to developer}.
+   content vary from developer to developer}. In the folders below, duplicate template files and remove ".template" from the names of the duplicates.
```

Comment:
Technically we are directly providing the files to be created, and not the folders.